### ISSUE TITLE
Cook doesn't cook meat

### ISSUE THREAD

**Issue by perecnicek** [ldtteam/minecolonies#4455]
### Minecolonies version
Version: minecolonies-0.10.763-BETA-universal

### Expected behavior

-  I expect that cook will give the request for food and deliverman brings him meat or potatoes and he will cook it.

### Actual behaviour

-  Cook request food for example raw chicken and deliverman bring him the chicken but cook doesn't put chicken into furnace and just waiting. That cousing stuck for cook and he dont want any other food like apple which dont need treatment in furnace. So my people are in restaurant and waiting until the death.

### Steps to reproduce the problem

Just play this version game and build cook

**Comment by SirSmuggler** [comment]
My cook is even worse of in this version. He won't request food of any kind. Neither will he cook food (steak f.ex.) if I give it to him. He will how ever feed the other citisens if I give him cooked food manually. He will even take cooked food out of his furnaces if I put it in ther manually.

**Comment by Tarumba** [comment]
Same for me, Cook doesn't request food. 
I can get him to cook raw food himself only if i give him a full stack of raw food items. everything works fine if I put the raw food in the furnace myself, even if it is not a full stack. Seems to be a problem with all the workers who use a furnace as i have the same problem with the smelter.

**Comment by jolandepoel** [comment]
Some things you could try: 
1) Have you set a fuel the cook could use on the second page of the hut options?
2) Try breaking the hut block and rebuilding the restaurant.
Edit: spelling

**Comment by perecnicek** [comment]
> Some things you could try:
> 
> 1. Have you set a fuel the cook could use on the second page of the hut options?
> 2. Try breaking the hut block and rebuilding the restaurant.
>    Edit: spelling

1. yes i set fuel for logs, coal and planks
2. I tried it and nothing the cook just put the fuel into furnace and he doesn't put meat/potatoes into furnace.

**Comment by perecnicek** [comment]
I made progress in solving this. The problem is not in the cook. Problem is that that citizens can't recognize smaller amount of items than stack (64) i found this in smeltery when smelter put 64 iron ore into furnace but he doesn' put the fuel because he had only 49 logs. When i gave him stack of logs he put it into furnace. I tried this with cook and it worked!. So problem is in recognize amount of items.
